app.py

```
@@ -1116,6 +1116,50 @@ async def insight(req: InsightRequest):
+        from core.rag.simple_rag import get_rag_stats
+
+        stats = get_rag_stats()
+        return {
+            "enabled": str(os.getenv("FEATURE_RAG", "")).strip().lower()
+            in {"1", "true", "on", "yes"},
+            "stats": stats,
+        }
+    except ImportError:
+        logger.warning("RAG module not available")
+        return {"enabled": False, "error": "RAG feature not available"}
+    except Exception as e:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)